src/main/java/org/broadinstitute/hellbender/engine/FeatureManager.java

```diff
@@ -248,29 +247,29 @@ void addToFeatureSources(final int featureQueryLookahead, final FeatureInput<? e
     }
 
     /**
-     * Given a Field known to be of type FeatureInput (or a Collection thereof), retrieves the type
+     * Given a ArgumentDefinition for an argument known to be of type FeatureInput (or a Collection thereof), retrieves the type
```

Comment:
```suggestion
     * Given an ArgumentDefinition for an argument known to be of type FeatureInput (or a Collection thereof), retrieves the type
```

Reply:
This commit is from another branch that was required for the new Barclay version, but thats been merged into master now and this is rebased on that, so this commit is gone.